root-sync/overlays/test/cert-manager/patch-issuer.yaml

```diff
@@ -0,0 +1,10 @@
+- op: add
+  path: /spec/acme/solvers/-
+  value:
+    selector: {}
+    dns01:
+      cloudDNS:
+        project: pht-01hsv4d2m0n
+- op: add
+  path: /spec/acme/email
+  value: vedant.thapa@phac-aspc.gc.ca
```

Comment:
**Non-blocking suggestion**: Does the team have a shared mailbox for cert management?

Reply:
Not that I'm aware of cause it's usually up to the project owners to decide how they'd like to manage TLS certificates.